README.md

```diff
@@ -111,6 +111,11 @@ The following describes the key directories that make up this repository.
 * `hack/`: scripts used for development and build processes
 * `test/`: scripts, configuration, and code used for end-to-end testing
 
+## Roadmap
+Please find linked the currently planned [roadmap](./ROADMAP.md) for Tanzu Community Edition. 
+
+
```

Comment:
Linting will fail here with too many spaces

Reply:
Weird that I don't see the lint checks failing https://github.com/vmware-tanzu/community-edition/pull/2415/checks

Reply:
Likely because of https://github.com/vmware-tanzu/community-edition/blob/25d1363334c8a6d8110c384c4aa209cb53a43e6c/README.md?plain=1#L1. But we should remove these extra lines regardless.

Reply:
Ah. Why disable markdown lint though? Seems like a full file disable

Reply:
I think that was unintentional. I'm addressing that here: https://github.com/vmware-tanzu/community-edition/pull/2509